src/cleaning_functions.py

```diff
@@ -24,26 +24,29 @@ def get_label_encoded_feature(feature):
     return [label_encoded_feature]
 
 
-def get_one_hot_encoded_feature(feature):
+def get_one_hot_encoded_feature(feature, feature_name):
```

Comment:
imho the feature name should not be added here, but later when actually saving the thing to a file
this is on the edge of violating design by contract principle and single responsibility principle..